client/pom.xml

```diff
@@ -42,6 +42,11 @@
             <version>2.29.1</version>
             <scope>provided</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-text</artifactId>
+            <version>1.10.0</version>
+        </dependency>
```

Comment:
can we do this without adding a new dependency? like just use `StringBuilder` and append `\n` at the end

Reply:
We can do that, I wanted to avoid using `\n` and using an inbuilt line append and hence added the new deps. Will change.